cypress/integration/proposal_administration.ts

```diff
@@ -162,7 +168,11 @@ context('Proposal administration tests', () => {
         );
     });
 
-    it('Should be able to re-open proposal for submission', () => {
+    it('Should be able to re-open proposal for submission', function () {
+      // if (!featureFlags.getEnabledFeatures().get(FeatureId.EXTERNAL_AUTH)) {
+      //   //temporaily disable test in for stfc until logout function works
+      //   this.skip();
+      // }
```

Comment:
Spoken to @vyshnavi1605 about these commented lines, will re-review once these are removed